### PR TITLE
Use default value from `quarkus.http.read-timeout` in RESTEasy Reactive

### PR DESCRIPTION
**Pull request by Sgitario**
Relates https://github.com/quarkusio/quarkus/issues/32992

### CODE REVIEW

File: independent-projects/resteasy-reactive/server/vertx/src/main/java/org/jboss/resteasy/reactive/server/vertx/VertxResteasyReactiveRequestContext.java [hunk 1]
```diff
@@ -240,7 +240,7 @@ public InputStream createInputStream() {
             context.getBody().getBytes(data);
             return new ByteArrayInputStream(data);
         }
-        return new VertxInputStream(context, 10000, this);
+        return new VertxInputStream(context, getDeployment().getRuntimeConfiguration().readTimeout().toMillis(), this);
```
Comment (Sgitario) [inline independent-projects/resteasy-reactive/server/vertx/src/main/java/org/jboss/resteasy/reactive/server/vertx/VertxResteasyReactiveRequestContext.java:243]: This was already being used when copying an existing input stream here: https://github.com/quarkusio/quarkus/blob/fe1f9554fd2c49462acaf4a78e2d809a6bc22a82/independent-projects/resteasy-reactive/server/vertx/src/main/java/org/jboss/resteasy/reactive/server/vertx/VertxResteasyReactiveRequestContext.java#L232

**Review by geoand — APPROVED** [review]
Nice!

**Comment by Sgitario** [comment]
The JDK 17 failure is not related. Merging.

**Comment by naah69** [comment]
@geoand Will this pr merge to 2.16.x?

**Comment by geoand** [comment]
It is unlikely we will be doing any more `2.16` releases at this point.

**Comment by naah69** [comment]
> It is unlikely we will be doing any more `2.16` releases at this point.

Is there any LTS version in 2.x?

**Comment by geoand** [comment]
See [this](https://access.redhat.com/support/policy/updates/jboss_notes#p_quarkus).

Note that `2.13` which is a supported product, is going to be out of maintenance in a few months, so I would not suggest going down from `2.16`, but going up to the latest `3.x`.

**Comment by naah69** [comment]
@geoand
Yes,we will upgrade to 3.x in the next few months.

We have many service and common utils base on 2.x, so we should upgrade them on the high priority.

But we had many exception about https://github.com/quarkusio/quarkus/issues/32992 now.
Is there any way to fix it if the pr won't merge to 2.x.

**Comment by geoand** [comment]
> Is there any way to fix it if the pr won't merge to 2.x.

It wasn't really clear what the problem was in the first place. Upgrading to 3.x is the way forward

**Comment by naah69** [comment]
@geoand 

Upgraing to 3.x is  long term plan.

What is emergency measures that is reslove the problem that is readtimeout of VertxInputStream?

**Comment by geoand** [comment]
Nothing I can think of

**Pull request merged**